### DISCUSSION TITLE
Animated HEIC Files

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7752] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I make lossless animated HEIC files out of PNG/JPG image sequences?

**Comment by superuser-miguel** [comment]
I'm not on Windows, but have you tried something like `magick convert -quality 100 -define heic:lossless=true image1.png image2.png image3.png output.heic`

**Reply by fmw42** [reply]
On IM 7, use just magick, not magick convert.  Be sure that you have installed the latest libheif library and recompiled imagemagick.

**Reply by Jay123210599** [reply]
Where do I install the libheif library? How do I recompile imagemagick?

**Reply by fmw42** [reply]
Sorry a Windows user will have to explain.

**Comment by snibgo** [comment]
This has come up in the past, but I can't find the posts.

As far as I remember, to write HEIC files IM needs the libx265 library. That has licensing restrictions which means the pre-built IM systems on Windows can't be distributed with libx265, and hence can't write HEIC files.

**Reply by Jay123210599** [reply]
So I need to build it from the source?
